server/src/routes/rootRouter.js

```diff
@@ -3,11 +3,14 @@ import userSessionsRouter from "./api/v1/userSessionsRouter.js"
 import usersRouter from "./api/v1/usersRouter.js"
 import clientRouter from "./clientRouter.js"
 import trailsRouter from "./api/v1/trailsRouter.js"
+import reviewRouter from "./api/v1/reviewRouter.js"
+
 const rootRouter = new express.Router()
 rootRouter.use("/", clientRouter)
 
 rootRouter.use("/api/v1/user-sessions", userSessionsRouter)
 rootRouter.use("/api/v1/users", usersRouter) //place your server-side routes here
 rootRouter.use("/api/v1/trails", trailsRouter)
+rootRouter.use("/api/v1/reviews", reviewRouter)
```

Comment:
This should be a nested route for reviews that belong to a trail

Reply:
DONE